File: languages/java/src/main/java/com/bloock/sdk/entity/identity_v2/IssuerParams.java
package com.bloock.sdk.entity.identity_v2;

import com.bloock.sdk.bridge.proto.IdentityEntitiesV2;

public class IssuerParams {
  private final Method method;
  private final Blockchain blockchain;
  private final Network network;

  public IssuerParams() {
    this.method = Method.POLYGONID;
    this.blockchain = Blockchain.POLYGON;
    this.network = Network.MUMBAI;
  }

  public IssuerParams(Method method, Blockchain blockchain, Network network) {
    this.method = method;
    this.blockchain = blockchain;
    this.network = network;
  }

  public IdentityEntitiesV2.IssuerParams toProto() {
    return IdentityEntitiesV2.IssuerParams.newBuilder()
        .setMethod(this.method.toProto())
        .setBlockchain(this.blockchain.toProto())
        .setNetworkId(this.network.toProto())
        .build();
  }
}

File: languages/php/Bloock/Entity/IdentityV2/IssuerParams.php
<?php

namespace Bloock\Entity\IdentityV2;

class IssuerParams
{
    private string $method;
    private string $blockchain;
    private string $network;

    public function __construct(string $method = null, string $blockchain = null, string $network = null)
    {
        $this->method = $method;
        $this->blockchain = $blockchain;
        $this->network = $network;
    }

    public function toProto(): \Bloock\IssuerParams
    {
        $s = new \Bloock\IssuerParams();
        $s->setMethod(Method::toProto($this->method));
        $s->setBlockchain(Blockchain::toProto($this->blockchain));
        $s->setNetworkId(Network::toProto($this->network));

        return $s;
    }
}
